smartsim/settings/launch_settings.py

```diff
@@ -126,6 +126,15 @@ def __init__(
             """The launcher type"""
         except ValueError:
             raise ValueError(f"Invalid launcher type: {launcher}")
+
+        if launch_args:
```

Comment:
Ditto here: Can we make this an explicit nullish check?
```suggestion
        if launch_args is not None:
```

Reply:
changed